tools/read_index_ids/main/main.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import "fmt"
+
+import (
+	"os"
+	"time"
+
+	"github.com/m3db/m3db/persist/fs"
+	"github.com/m3db/m3db/ts"
+	xlog "github.com/m3db/m3x/log"
+	"github.com/m3db/m3x/pool"
+	"github.com/pborman/getopt"
+)
+
+const (
+	defaultBufferReadSize  = 10
+	defaultBufferCapacity  = 1024 * 1024 * 1024
+	defaultBufferPoolCount = 10
+)
+
+func main() {
+	var (
+		optPathPrefix = getopt.StringLong("path-prefix", 'p', "", "Path prefix [e.g. /var/lib/m3db]")
+		optNamespace  = getopt.StringLong("namespace", 'n', "", "Namespace [e.g. metrics]")
+		optShard      = getopt.Uint32Long("shard-id", 's', 0, "Shard ID [expected format uint32]")
+		optBlockstart = getopt.Int64Long("block-start", 'b', 0, "Block Start Time [in nsec]")
+		log           = xlog.NewLogger(os.Stderr)
+	)
+	getopt.Parse()
+
+	if *optPathPrefix == "" ||
+		*optNamespace == "" ||
+		*optShard < 0 ||
+		*optBlockstart <= 0 {
+		getopt.Usage()
+		os.Exit(1)
+	}
+
+	seeker := fs.NewSeeker(*optPathPrefix, defaultBufferReadSize, pool.NewBytesPool(
+		[]pool.Bucket{pool.Bucket{Capacity: defaultBufferCapacity, Count: defaultBufferPoolCount}}, nil))
+
+	err := seeker.Open(ts.StringID(*optNamespace), uint32(*optShard), time.Unix(0, int64(*optBlockstart)))
+
+	if err != nil {
+		log.Fatalf("Unable to open file: %v", err)
+	}
+
+	fileIds := seeker.IDs()
+	if fileIds == nil {
+		log.Fatalf("Unable to retrieve ids")
```

Comment:
Minor minor nit: all our log messages currently begin with lower case, this seems to be something of a standard common practice in other Go services/applications too, might be worth conforming these log statements too.

Also for this specific log message, perhaps "no ids in index" is more correct?  If the seeker opened correctly then the file format and the header structures are all valid, there's just no IDs in the file itself.